CONTRIBUTING.rst

```diff
@@ -0,0 +1,197 @@
+Contributing to the Jupyter NbClassic
+====================================
+
+If you're reading this section, you're probably interested in contributing to
+Jupyter.  Welcome and thanks for your interest in contributing!
+
+Please take a look at the Contributor documentation, familiarize yourself with
+using the Jupyter NbClassic, and introduce yourself on the mailing list and
+share what area of the project you are interested in working on.
+
+General Guidelines
+------------------
+
+For general documentation about contributing to Jupyter projects, see the
+`Project Jupyter Contributor Documentation`__.
+
+__ https://jupyter.readthedocs.io/en/latest/contributing/content-contributor.html
+
+
+Setting Up a Development Environment
+------------------------------------
+
+Installing Node.js and npm
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Building the NbClassic from its GitHub source code requires some tools to
+create and minify JavaScript components and the CSS,
+specifically Node.js and Node's package manager, ``npm``.
+It should be node version ≥ 6.0.
+
+If you use ``conda``, you can get them with::
+
+    conda install -c conda-forge nodejs
+
+If you use `Homebrew <https://brew.sh/>`_ on Mac OS X::
+
+    brew install node
+
+Installation on Linux may vary, but be aware that the `nodejs` or `npm` packages
+included in the system package repository may be too old to work properly.
+
+You can also use the installer from the `Node.js website <https://nodejs.org>`_.
+
+
+Installing the Jupyter NbClassic
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you have installed the dependencies mentioned above, use the following
+steps::
+
+    pip install --upgrade setuptools pip
+    git clone https://github.com/jupyter/nbclassic
+    cd notebook
+    pip install -e .
+
+If you are using a system-wide Python installation and you only want to install the notebook for you,
+you can add ``--user`` to the install commands.
+
+Once you have done this, you can launch the master branch of Jupyter notebook
+from any directory in your system with::
+
+    jupyter nbclassic
+
+Verification
+^^^^^^^^^^^^
+
+While running the notebook, select one of your notebook files (the file will have the extension ``.ipynb``).
+In the top tab you will click on "Help" and then click on "About". In the pop window you will see information about the version of Jupyter that you are running. You will see "The version of the notebook server is:".
```

Comment:
> "The version of the notebook server is:"

Is this message still correct? Do we need to change it?

Reply:
I still it it still correct. We have not changed that behavior while moving from notebook to nbclassic.